### ISSUE TITLE
Bug: Fix playing from mobile links SoundCloud app

### ISSUE THREAD

**Issue by Perdition0** [eritislami/evobot#328]
Hi there,

For some reason evobot doesn’t recognize soundcloud.app.goo.gl, would you please be able to implement a fix? :)
Thanks you

**Comment by evrardco** [comment]
I was able to reproduce that bug.
Steps to reproduce:

 1. Clone the repo and launch the bot using docker
 2. Pick a sound on soundclound using the mobile app (I used the android one)
 3. Try to play it using the following command: "/play <your-mobile-link>"
 4. The bot gives the following error in the chat: "Cannot read property 'url' of undefined". Instead of playing the song.

Details:

- The song I used was part of a playlist.

- Mobile link: https://soundcloud.app.goo.gl/c39H

- Desktop link: https://soundcloud.com/yourparadis/sets/jazzhip7c

- The desktop link worked as expected, unlike the mobile one.
